crates/ironrdp-web/src/session.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(non_snake_case)]
+
```

Comment:
question: I don’t have any warning when I open this file or when I try to compile manually. Do you understand why the warning is emitted on your side?

suggestion: Add a comment explaining why the lint is silenced. We don’t really want to silent that forever, so at some point it would be nice to remember we didn’t intend to so in the first place.

Reply:
I added a link to an issue I opened for wasm-bindgen.